.ruff.toml

```diff
@@ -0,0 +1,13 @@
+# Default settings for Ruff in CPython
+
+# PYTHON_FOR_REGEN
+target-version = "py310"
+
+# PEP 8
+line-length = 79
+
+# Nicer diagnostic output
+output-format = "full"
```

Comment:
I think this is the default nowadays?

```suggestion
```

Reply:
For reference: https://docs.astral.sh/ruff/settings/#output-format